cypress/integration/Profile Image Upload Tests/profileImageUpload.specs.js

```diff
@@ -0,0 +1,65 @@
+ /// <reference types="cypress" />
+ import {profileTab} from '../../utils/profileTab'
+
+ describe("NeetoAuth Profile Image upload functionality", () => {
+   beforeEach(() => {
+    profileTab();
+  });
+    
+  it("Uploading image less than 5MB", () => {
+      cy.uploadImg('lessThan5MB.jpg','Profile image successfully updated!');
+  });
+
+  it("Uploading image more than 5MB", () => {
+      cy.uploadImg('moreThan5MB.jpg','Something went wrong.');
+  });
+
+    it("Uploading unallowed file type (.pdf)", () => {
+      cy.uploadImg('wrongFileFormat.pdf','Something went wrong.');
+  });
+
+  it("Uploading multiple images file", () => {
+      cy.get('[data-cy="profile-image-upload-file-field"]').attachFile('./lessThan5MB').attachFile('./lessThan5MB');
+      cy.get('[data-cy=toastr-message-container]').eq(0).should('have.text','Something went wrong.')
+      cy.get('[data-cy=toastr-message-container]').eq(1).should('have.text','Something went wrong.')
```

Comment:
Also, in this test only one file will be attached. We can assert that and change the description of our 'it' block.